### ISSUE TITLE
Jak moc to bereme vážně?

### ISSUE THREAD

**Issue by vairad** [denitara/esence#1]
Tak jsem se tím v rychlosti prokousal...
Teď je tam všechno vložené bez erroru. (Nejzákeřnější byl v konstruktoru osoby, kde jakoby vypadla jedna hodnota a byly tam dvě čárky za sebou.)

Potom jsem zkoušel přemluvit PDO, aby z něj padaly chyby a bohužel se mi to nepodařilo, protože jsem nepochopil zanoření, dědění, těch tříd co tam jsou na práci s db...

Takže má otázka zní jak moc vážně to bereš a jak moc to chceš řešit sama případně jak moc se do toho mám zapojit taky.

**Comment by denitara** [comment]
No, ta databáze by měla být funkční, to je teď asi nejdůležitější... Samozřejmě Tě tím nechci zatěžovat, nevím, jestli to někdo další od nás umí (možná, že jo, ale nevím o tom), takže se to buď pokusím zprovoznit sama nebo budu ráda za Tvou pomoc :-) Každopádně už teď Ti moc děkuju :-)
